### PR TITLE
修复 Scale9Sprite 显示合图时的边缘黑线

### PR DESCRIPTION
**Pull request by pandamicro**
Re: cocos-creator/fireball#3675

Changes proposed in this pull request:
- 修复 Scale9Sprite 显示合图时的边缘黑线
- 在 UC 上关闭 WebGL
- 修复 touch start point 被错误更新的问题
- 更新 JSB downloaders
- 修复 JSB 上 Texture2D 的 pixelWidth 和 pixelHeight 属性问题

> Makes sure these boxes are checked before submitting your PR - thank you!
> - [x] If your pull request has gone "stale", you should **rebase** your work on top of the latest version of the upstream branch.
> - [x] If your commit history is full of small, unimportant commits (such as "fix pep8" or "update tests"), **squash** your commits down to a few, or one, discreet changesets before submitting a pull request.
> - To official teams:
>   - [x] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) and end files with a newline
>   - [x] Document new code with comments in source code based on [API Docs](https://github.com/cocos-creator/fireball#api-docs)

@cocos-creator/engine-admins

### CODE REVIEW

File: jsb/jsb-loader.js [hunk 1]
```diff
@@ -63,13 +63,31 @@ cc.loader.addDownloadHandlers({
     'mp4' : downloadAudio,
     'm4a' : downloadAudio,
 
+    // Txt
+    'txt' : empty,
```
Comment (jareguo) [inline jsb/jsb-loader.js:67]: 这些 empty 的会怎么加载？

Comment (pandamicro) [inline jsb/jsb-loader.js:67]: 不加载，直接取，用 web downloader 中的方式会报错

File: cocos2d/core/sprites/CCScale9Sprite.js [hunk 2]
```diff
@@ -162,22 +163,24 @@ var simpleQuadGenerator = {
 
         //uv computation should take spritesheet into account.
         var l, b, r, t;
+        var texelCorrectX = FIX_ARTIFACTS_BY_STRECHING_TEXEL ? 0.5 / atlasWidth : 0;
```
Comment (jareguo) [inline cocos2d/core/sprites/CCScale9Sprite.js]: 其实这里除以 atlasWidth 是多余的，只要下面写 `l = (textureRect.x + texelCorrectX) / atlasWidth;`

Comment (jareguo) [inline cocos2d/core/sprites/CCScale9Sprite.js]: 然后 texelCorrectX 和 texelCorrectY 就能合并成一个变量

**Pull request merged**